kolibri/plugins/facility/assets/src/views/DataPage/SyncInterface/index.vue

```diff
@@ -266,6 +268,10 @@
         context:
           "Describes the 'sync' button which is used to synchronize data from a facility with the project on the Kolibri Data Portal.",
       },
+      manage_sync: {
```

Comment:
It seems we have lost our linting rule that enforced camelCase message ids, but I would prefer if we maintain this convention: `manageSync`.

Also maybe this should go into coreStrings, as this is needed elsewhere too?